### ISSUE TITLE
Feature: SSL Support

### ISSUE THREAD

**Issue by tlf30** [tlf30/monkey-netty#5]
It would be great to have SSL support, especially for multiplayer games that require user login, or pass sensitive information. 

I am not yet sure how it would work on UDP, but very doable on TCP. 
.

**Comment by oxplay2** [comment]
yes, i agree.

Netty already provide SSL support for TCP.

We just need to add possibility to use it :)

see for example:
https://github.com/netty/netty/blob/4.1/example/src/main/java/io/netty/example/objectecho/ObjectEchoClient.java


```
        final SslContext sslCtx;
        if (SSL) {
            sslCtx = SslContextBuilder.forClient()
                .trustManager(InsecureTrustManagerFactory.INSTANCE).build();
        } else {
            sslCtx = null;
        }
```

On server side can provide SelfSignedFactory or own certificate (if have).

Question is, how to allow user use it cia Monkey-Netty wrapper.

**Comment by tlf30** [comment]
Closed as #6 is merged.
